app/models/notification.rb

```
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 class Notification < ActiveRecord::Base
+  default_scope { order(:read, created_at: :desc) }
```

be careful with default scope, it could "hide" behavior